File: cpp/3110.score_of_a_string.cpp
#include <string>

class Solution {
 public:
    int scoreOfString(std::string s) {
        int i = 1;
        int result = 0;

        while (i < s.size()) {
            result +=
                std::abs(static_cast<int>(s[i - 1]) - static_cast<int>(s[i]));
            ++i;
        }
        return result;
    }
};

File: python/3110.score_of_a_string.py
class Solution:
    def scoreOfString(self, s: str) -> int:

        left = 0
        right = 1
        result = 0

        while right < len(s):
            result += abs(ord(s[left]) - ord(s[right]))
            left += 1
            right += 1
        return result
